src/test/com/wolfssl/test/WolfSSLSessionTest.java

```diff
@@ -2804,5 +2807,1334 @@ public Void call() throws Exception {
+
+            /* Enable CID */
+            int ret = sess.dtlsCidUse();
+            if (ret == WolfSSL.NOT_COMPILED_IN) {
+                System.out.println("\t\t... skipped (DTLS CID not " +
+                                   "compiled in)");
+                return;
+            }
+
+            /* Test null CID - should return BAD_FUNC_ARG */
+            ret = sess.dtlsCidSet(null);
+            assertEquals("Null CID should return BAD_FUNC_ARG",
+                         WolfSSL.BAD_FUNC_ARG, ret);
+
+            /* Test multiple calls to dtlsCidSet (overwrite scenario) */
+            byte[] firstCid = new byte[]{0x01, 0x02, 0x03};
+            byte[] secondCid = new byte[]{0x04, 0x05, 0x06, 0x07};
+
+            ret = sess.dtlsCidSet(firstCid);
+            if (ret == WolfSSL.SSL_SUCCESS) {
+                /* Verify first CID was set */
+                byte[] retrieved = sess.dtlsCidGetRx();
+                assertNotNull("First CID should be retrievable",
+                              retrieved);
+                assertArrayEquals("First CID should match", firstCid,
+                                  retrieved);
+
+                /* Overwrite with second CID */
+                ret = sess.dtlsCidSet(secondCid);
+                assertNotNull("Second dtlsCidSet should return a value",
+                              Integer.valueOf(ret));
+
+                if (ret == WolfSSL.SSL_SUCCESS) {
+                    /* Verify second CID overwrote first */
+                    retrieved = sess.dtlsCidGetRx();
+                    assertNotNull("Second CID should be retrievable",
+                                  retrieved);
+                    assertArrayEquals("Second CID should have " +
+                                      "replaced first",
+                                      secondCid, retrieved);
+                    assertEquals("RX size should match second CID",
+                                 secondCid.length,
+                                 sess.dtlsCidGetRxSize());
+                }
+            }
+
+            /* Test calling CID methods without dtlsCidUse() */
+            WolfSSLSession sess2 = new WolfSSLSession(dtlsCtx);
+            try {
+                /* Don't call dtlsCidUse(), try to set CID anyway */
+                byte[] testCid = new byte[]{0x11, 0x22};
+                ret = sess2.dtlsCidSet(testCid);
+                /* Should fail or return error since CID not enabled */
+                assertNotEquals("Setting CID without dtlsCidUse should fail",
+                                WolfSSL.SSL_SUCCESS, ret);
+
+                /* Check that CID is not enabled */
+                int enabled = sess2.dtlsCidIsEnabled();
+                assertTrue("CID should not be enabled", enabled == 0 ||
+                           enabled == WolfSSL.NOT_COMPILED_IN);
+
+            } finally {
+                sess2.freeSSL();
+            }
+
+            System.out.println("\t\t... passed");
```

Comment:
Looks like this needs to get backed up one `\t` so it lines up with the other "... passed" column items. The skipped or failed cases will also need adjusted if they use the same number of `\t`'s.